src/pivotal-ui/components/meta/need-help-orgs.hbs

```diff
@@ -2,7 +2,7 @@
 
   <div class="row">
     <div class="col-xs-7 col-xs-offset-1 txt-c">
-      <img alt="support avatar" class="avatar" height="60" src="styleguide/orgs-svgeople-avatar.svg" width="60">
+      {{svg "@npmcorp/pui-css-meta/orgs-svgeople-avatar.svg" alt="support avatar" class="avatar" height="60" width="60"}}
```

Comment:
We decided that using an inline svg was overkill.

Turns out that it is not the case when we're using `fromPackage`